CONTRIBUTING.md

```diff
@@ -52,3 +52,7 @@ To build the solution you can:
 
 1. Open the solution `Src\RE2.Native.sln` and build.
 2. Open the solution `Src\SarifPatternMatcher.sln` and build.
+
+## Troubleshooting First Commit
```

Comment:
>Troubleshooting [](http://example.com/codeflow?start=3&length=15)

Nice work! A couple things, first, we are inconsistent in the naming of this document between projects. That's not your problem, though, we can fix that later. :)

I have some other comments I will offer offline. Let's get this in and we can iterate!!